app/processors/audio_processor.rb

```diff
@@ -132,7 +132,7 @@ def slice_audio
     source_wav = get_wav_from_source
     task_tmp = audio_monster.create_temp_file(File.basename(source.path))
     finish = options['finish'].blank? ? '' : "=#{options['finish']}"
-    audio_monster.slice_wav(source_wav.path, task_tmp.path, options['start'], (options['length'] || finish))
+    audio_monster.slice_wav(source_wav.path, task_tmp.path, options['start'].to_i, (options['length'] || finish).to_i)
```

Comment:
Line is too long. [118/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.